utils.groovy

```diff
@@ -92,6 +92,12 @@ def validate_pipecfg(pipecfg, is_hotfix) {
     }
 }
 
+def load_gc() {
+    // Read YAML file
```

Comment:
This needs to follow a similar logic as `load_pipecfg()`, except that in the non-"in-tree" case, let's not clone again, but verify that the `pipecfg` directory is already there and if so, load it from there. Also shouldn't care about hotfix-related bits.